Tests/Unit/MSFT_xIISFeatureDelegation.tests.ps1

```diff
@@ -4,14 +4,30 @@ $global:DSCResourceName = 'MSFT_xIISFeatureDelegation'
 #region HEADER
 
 [String] $moduleRoot = Split-Path -Parent (Split-Path -Parent (Split-Path -Parent $Script:MyInvocation.MyCommand.Path))
- if ( (-not (Test-Path -Path (Join-Path -Path $moduleRoot -ChildPath 'DSCResource.Tests'))) -or `
-      (-not (Test-Path -Path (Join-Path -Path $moduleRoot -ChildPath 'DSCResource.Tests\TestHelper.psm1'))) )
-{
-    & git @('clone','https://github.com/PowerShell/DscResource.Tests.git',(Join-Path -Path $moduleRoot -ChildPath '\DSCResource.Tests\'))
+$repoSource = (Get-Module -Name $global:DSCModuleName -ListAvailable)
+
+if ($repoSource.RepositorySourceLocation.Host -eq 'www.powershellgallery.com') {
+    if ( -not (Test-Path -Path (Join-Path -Path $moduleRoot -ChildPath 'DscResourceTestHelper')) ) {
+        #install test folders from gallery
+        Save-Module -Name 'DscResourceTestHelper' -Path $moduleRoot
+    }
+
+    $testModuleVer = '0.2.0.0'
```

Comment:
It doesn't strike me as an awesome idea to hard code this, could we detect the version on save? Or do something to 'discover' the version number instead of hard coding it?